src/monster_event.py

```diff
@@ -3,45 +3,43 @@
 """
 from monster import Monster
 
-from collections import namedtuple
-Equipment = namedtuple('Equipment', ('name', 'strength'))
-Item = namedtuple('Item', ('name', 'heal_value'))
-
 
 class MonsterEvent:
     """
-    Interfaces class for Monster Event
+    This class defines all Events related to Monster
     """
 
-    def __init__(self, monster, screen):
+    def __init__(self, monster, player, screen):
         self.screen = screen
         self.top = "--- Monster-Event ---"
         self.header = [
             "Ein " + monster.name + " " + monster.str + " greift an."
             ]
         Monster.fight(monster, player, self)
```

Comment:
Warum soll hier ein Kampf ausgefuehrt werden? Davon abgesehen ist das hier das gleiche wie in der anderen Klasse, so sollte die Methode nicht ausgefuehrt werden.